app/helpers/structured_data_helper.rb

```diff
@@ -18,6 +18,25 @@ def structured_data(type, data)
     end
   end
 
+  def blog_structured_data(page)
+    frontmatter = page.frontmatter
+
+    data = {
+      headline: frontmatter[:title],
+      image: frontmatter[:images].values.map { |h| h["path"] },
+      datePublished: frontmatter[:date],
+      keywords: frontmatter[:keywords],
+      author: [
+        {
+          "@type": "Organization",
```

Comment:
Not sure if it matters, but could the author type be conditional (Organisation or Person) based on whether the blog author has defaulted to "Get into Teaching"? https://github.com/DFE-Digital/get-into-teaching-app/blob/dc56f963d24db8d41acbc5e7a8d989c08cb32d2c/app/views/blog/_date_and_author.html.erb#L6

Reply:
Yeah that's a good shout, I had thought all the blog posts were `author: Get Into Teaching` but some are missing the key entirely and others are actual names. I'll clarify it with the content team and update the PR.